### PR TITLE
Adds function to get covariate balance for plotting or tabulation

### PR DESCRIPTION
**Pull request by gowthamrao**
Part of https://github.com/OHDSI/SkeletonComparativeEffectStudy/issues/27

### CODE REVIEW

**Review by schuemie — APPROVED** [review]

**Comment by schuemie** [comment]
@gowthamrao : could you resolve the conflict? It seems git is mixing up the new power and covariate balance functions!

**Comment by gowthamrao** [comment]
@schuemie i think i resolved it - and tested. please merge

File: R/DataPulls.R [hunk 1]
```diff
@@ -214,6 +214,107 @@
...
+                          covariate = NULL,
+                          databaseSchema,
+                          databaseId,
+                          analysisId) {
+  if (!is.null(connection)) {
+    sql <-   "SELECT *
+              FROM @databaseSchema.covariate
+              WHERE database_id = '@databaseId'
+              and analysis_id = @analysisId;"
+    sql <-
```
Comment (schuemie) [inline R/DataPulls.R]: Why a newline here?

**Comment by schuemie** [comment]
Looks good, except you're inconsistent in use of newlines. I suggest we just never put a newline right after `<-`.

File: R/DataPulls.R [hunk 2]
```diff
@@ -214,6 +214,104 @@
...
+    interactionCovariateIdNull <- is.null(interactionCovariateId)
+    sql <-   "SELECT *
+              FROM @databaseSchema.covariate_balance
+              WHERE target_id = @targetId
+              	and comparator_id = @comparatorId
+              	and outcome_id = @outcomeId
+              	and database_id = '@databaseId'
+              	{@interactionCovariateIdNull} ? {and interaction_covariate_id IS NULL}:{and interaction_covariate_id = @interactionCovariateId}
+              	and analysis_id = @analysisId;"
+    sql <-
```
Comment (schuemie) [inline R/DataPulls.R]: Some more left

**Review by schuemie — APPROVED** [review]

**Pull request merged**